subprojects/docs/src/docs/userguide/working_with_files.adoc

```diff
@@ -150,6 +150,8 @@ Notice how we specify the destination and name of the archive instead of an `int
 
 Each type of archive has its own task type, the most common ones being link:{groovyDslPath}/org.gradle.api.tasks.bundling.Zip.html[Zip], link:{groovyDslPath}/org.gradle.api.tasks.bundling.Tar.html[Tar] and link:{groovyDslPath}/org.gradle.api.tasks.bundling.Jar.html[Jar]. They all share most of the configuration options of `Copy`, including filtering and renaming.
 
+One notable difference between the `Copy` and `Sync` tasks and the archive tasks is that duplicate files are not allowed in the latter by default.
```

Comment:
This comment is still unresolved: https://github.com/gradle/gradle/pull/10143#discussion_r310093542